### ISSUE TITLE
Shallow clone detection (or depth)

### ISSUE THREAD

**Issue by giarve** [OCA/oca-port#56]
There are many times when we perform a shallow clone, and later we use the tool without realizing we are only taking N last commits but not the whole commit history since when the module was added.

Maybe a confirmation alert (answer y/n) would be better than performing a full clone without asking first.

**Comment by sebalix** [comment]
Good point. Should we block fully the use of the tool on such cloned repository? A warning+confirmation is probably not enough if we want to prevent the opening of PRs on OCA with limited commits history.
